staking/stakingprecompile.md

```diff
@@ -0,0 +1,95 @@
+ - `nominate(address collator, uint256 amount)` - if the caller is not a nominator, this function adds them to the set of nominators. If the caller is already a nominator, then it adjusts their nomination amount
+ - `leave_nominators()` - leave the set of nominators and revoke all ongoing nominations
+ - `revoke_nominations(address collator)` - revoke a specific nomination
+ - `nominator_bond_more(address collator, uint256 more)` - nominator increases bond to a collator by specified amount
+ - `nominator_bond_less(address collator, uint256 less` - nominator decreases bond to a collator by specified amount
+
+## Interacting with the Staking Precompile
+
+### Checking Prerequisites
+
+ - Have MetaMask installed and [connected to Moonbase Alpha](/getting-started/moonbase/metamask/)
+ - Have an account with over {{networks.moonbase.staking.min_nom_stake}} tokens. You can get this from [Mission Control](/getting-started/moonbase/faucet/)
+
+### Remix Set Up
+1. Get a copy of the [latest staking interface](https://github.com/PureStake/moonbeam/blob/master/runtime/precompiles/src/StakingInterface.sol)
+2. Copy and paste the file contents into a Remix file named StakingInterface.sol
+
+![Copying and Pasting the Staking Interface into Remix](/images/staking/stakinginterface1.png)
+
+### Compile the Contract
+1. Click on the Compile Tab, second from top
+2. Compile [Staking Interface.sol](https://github.com/PureStake/moonbeam/blob/master/runtime/precompiles/src/StakingInterface.sol)
+
+![Compiling StakingInteface.sol](/images/staking/compile4.png)
+
+### Access the Contract
+1. Click on the Deploy and Run Tab, directly below the Compile Tab in Remix
+2. Make sure "Injected Web3" is selected in the Environment drop down
+3. Make sure “ParachainStaking - StakingInterface.sol” is selected in the Contract dropdown. Since this is a precompiled contract on Moonbase Alpha there is no need to deploy, instead we are going to provide the address of the precompile in the “AtAddress” Field
+4. Provide the address of the Staking precompile: `{{networks.moonbase.staking.precompile_address}}` and Click “At Address”
+
+![Provide the addressl](/images/staking/deploy1export.png)
+
+### Nominate a Collator
+For this example, we are going to be nominating a collator. Nominators are token holders who stake tokens, vouching for specific collators. Any user that holds a minimum amount of {{networks.moonbase.staking.min_nom_stake}} tokens as free balance can become a nominator. 
+
+1. Expand the panel with the contract address and find the nominate function and expand the panel to see the parameters.
+2. Provide the address of a collator such as `{{ networks.moonbase.staking.collators.address1 }}`
+3. Provide the amount to nominate in WEI. There is a minimum of {{networks.moonbase.staking.min_nom_stake}} DEV to nominate, so the lowest amount in WEI is `5000000000000000000`
+4. Press Transact and Confirm the Transaction in Metamask
```

Comment:
```suggestion
4. Press "transact" and confirm the transaction in Metamask
```

Reply:
I would maybe use same capitalization as the actual button or text in remix 🤷‍♀️